samza-core/src/main/java/org/apache/samza/storage/ChangelogPartitionManager.java

```diff
@@ -39,10 +48,25 @@
   private static final Logger log = LoggerFactory.getLogger(ChangelogPartitionManager.class);
   private boolean isCoordinatorConsumerRegistered = false;
 
-  public ChangelogPartitionManager(CoordinatorStreamSystemProducer coordinatorStreamProducer,
-      CoordinatorStreamSystemConsumer coordinatorStreamConsumer,
-      String source) {
-    super(coordinatorStreamProducer, coordinatorStreamConsumer, source);
+  /**
+   * Construct Changelog manager without a coordinator stream.
+   *
+   *  @param source Name of caller.
```

Comment:
What is the source used for?

Reply:
Source is a member of the parent class and it looks like it's mostly used for logging, except for some non-Kafka consumer/producer implementations.